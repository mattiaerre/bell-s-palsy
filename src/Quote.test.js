import { render } from '@testing-library/react';
import Quote from './Quote';

test('snapshot', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment()).toMatchSnapshot();
});
