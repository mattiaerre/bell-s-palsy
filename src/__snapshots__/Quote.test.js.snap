// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <figure>
    <blockquote
      cite="https://www.amazon.com/gp/product/1935209892/ref=as_li_qf_asin_il_tl"
      class="Quote"
    >
      <p>
        When you are a young person, you are like a young creek, and you meet many rocks, many obstacles and difficulties on your way. You hurry to get past these obstacles and get to the ocean.
      </p>
      <p>
        But as the creek moves down through the fields, it becomes larges and calmer and it can enjoy the reflection of the sky. It's wonderful. You will arrive at the sea anyway so enjoy the journey. Enjoy the sunshine, the sunset, the moon, the birds, the trees, and the many beauties along the way. Taste every moment of your daily life.
      </p>
    </blockquote>
    <figcaption>
      â€”Thich Nhat Hanh, 
      <cite>
        Good Citizens: Creating Enlightened Society
      </cite>
    </figcaption>
  </figure>
</DocumentFragment>
`;
